activerecord/lib/active_record/relation/calculations.rb

```diff
@@ -153,23 +153,27 @@ def calculate(operation, column_name)
     # See also +ids+.
     #
     def pluck(*column_names)
-      column_names.map! do |column_name|
-        if column_name.is_a?(Symbol) && attribute_alias?(column_name)
-          attribute_alias(column_name)
-        else
-          column_name.to_s
+      if loaded?
+        @records.pluck(*column_names)
```

Comment:
Indent here flushes history and creates too much nesting when a lot of edge cases of a method handled at the beginning. Can we use return instead and leave indent as is?